### ISSUE TITLE
The site flashscore.com has been updated

### ISSUE THREAD

**Issue by grandluckyman** [msarnacki/flashscore-scraper#5]
Hey! I wanted to collect data, but I failed. The site has been updated and now the code does not work. I tried to change the code myself, but ran into some problems. I tried to restore the old version of the site through https://archive.org, where I was able to load the first page of the match, but I could not open the Statistics tab, so I could not adapt the code to the new site. I would be very grateful if you manage to update the code.

**Comment by djoker11** [comment]
Yes, this is exactly what I have been looking for for a long time

please help to make it work with the new website
